src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in

```diff
@@ -37,24 +37,33 @@ from madlib_keras_wrapper import convert_string_of_args_to_dict
 from madlib_keras_helper import CLASS_VALUES_COLNAME
 from madlib_keras_helper import KerasWeightsSerializer
 
-def predict(schema_madlib, model_table, test_table, id_col, model_arch_table,
-            model_arch_id, independent_varname, compile_params, output_table,
-            **kwargs):
-    module_name = 'madlib_keras_predict'
-    input_tbl_valid(test_table, module_name)
-    input_tbl_valid(model_arch_table, module_name)
-    output_tbl_valid(output_table, module_name)
-
-    # _validate_input_args(test_table, model_arch_table, output_table)
+MODULE_NAME = 'madlib_keras_predict'
+def predict(schema_madlib, model_table, test_table, id_col,
+            independent_varname, output_table, **kwargs):
+    model_summary_table = add_postfix(model_table, '_summary')
+    input_tbl_valid(model_table, MODULE_NAME)
```

Comment:
We should probably validate `model_table` before calling `add_postfix()` to avoid failure if the tablename is passed in predicate as NULL